### PR TITLE
[DOC-2715] Create SMP landing page

### PR DESCRIPTION
**Pull request by KimberlyFields**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by wei-harness — APPROVED** [review]
Looks good: https://644b08d6a4d62b23912c45eb--harness-developer.netlify.app/docs/self-managed-enterprise-edition

the only thing I noticed is that, seemed you've removed the the featured tutorial section:
```
      {featuredTutorials && featuredTutorials.length > 0 && (
        <>
          <div className={styles.sectionDivider}></div>
          <div className={styles.subSection}>
            <h3>Featured Tutorials</h3>
            <TutorialCard FeatureList={featuredTutorials} featuredCard={true} />
          </div>
        </>
      )}
```
      So are we no longer showing Featured Tutorials on Docs landing pages?

**Comment by KimberlyFields** [comment]
> Looks good: https://644b08d6a4d62b23912c45eb--harness-developer.netlify.app/docs/self-managed-enterprise-edition
> 
> the only thing I noticed is that, seemed you've removed the the featured tutorial section:
> 
> ```
>       {featuredTutorials && featuredTutorials.length > 0 && (
>         <>
>           <div className={styles.sectionDivider}></div>
>           <div className={styles.subSection}>
>             <h3>Featured Tutorials</h3>
>             <TutorialCard FeatureList={featuredTutorials} featuredCard={true} />
>           </div>
>         </>
>       )}
> ```
> 
> ```
>   So are we no longer showing Featured Tutorials on Docs landing pages?
> ```

We don't have any tutorials for SMP yet. Will add the featured tutorial section to the SMP landing page when we have some tutorials.

File: src/components/Docs/data/selfManagedEnterpriseEditionData.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      {
+        name: "Feature highlights",
+        description:
+          "",
+        list: [
+          {
+            title: "Monitoring",
+            module: MODULES.platform,
+            description:
+              "Learn how to receive alerts on metrics such as CPU, memory, and disk usage.",
```
Comment (brian-f-harness) [inline src/components/Docs/data/selfManagedEnterpriseEditionData.ts:76]:
```suggestion
              "Learn how to monitor the infrastructure of your installation.",
```

File: src/components/Docs/data/selfManagedEnterpriseEditionData.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      {
+        name: "Get started",
+        description:
+          "",
+        list: [
+          {
+            title: "Install using Helm",
+            module: MODULES.platform,
+            description:
+              "Learn how to use Helm to install Self-Managed Enterprise Edition.",
```
Comment (brian-f-harness) [inline src/components/Docs/data/selfManagedEnterpriseEditionData.ts:48]: Not sure if we normally use the definite article when we refer to the product?

File: src/components/Docs/data/selfManagedEnterpriseEditionData.ts (hunk 1 above)
Comment (brian-f-harness) [inline src/components/Docs/data/selfManagedEnterpriseEditionData.ts:76]: I'll be merging a new version of this topic today. Prateek changed the description to:
Monitor the infrastructure of your installation

PR: https://github.com/harness/developer-hub/pull/1524

File: src/components/Docs/data/selfManagedEnterpriseEditionData.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            module: MODULES.platform,
+            description:
+              "Learn how to receive alerts on metrics such as CPU, memory, and disk usage.",
+            link: "/docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem",
+          },
+          {
+            title: "Back up and restore",
+            module: MODULES.platform,
+            description:
+              "Learn how to back up and restore Harness Self-Managed Enterprise Edition.",
```
Comment (brian-f-harness) [inline src/components/Docs/data/selfManagedEnterpriseEditionData.ts:83]: Looks like we don't use the definite article here either. If we don't normally use it, please ignore my other comment. We do use Harness here before SMP. We may want to be consistent on line 48 and use Harness there too?

**Review by brian-f-harness — APPROVED** [review]
lgtm. Added a couple comments and suggestions.

**Pull request merged**